packages/phones/src/phone-service/PhoneService.ts

```diff
@@ -1,5 +1,10 @@
+import {
+  getAsYouType,
+  ParsedPhoneNumber,
+  parsePhoneNumber,
+} from 'awesome-phonenumber';
```

Comment:
We should not load it directly

Reply:
@doniyor2109 I can only lazy load parsePhoneNumber method, I think the rest getAsYouType used not much